gradle/witness/gradle-witness.gradle

```diff
@@ -20,6 +20,10 @@ dependencyVerification {
         'de.jensd:fontawesomefx-commons:5539bb3335ecb822dbf928546f57766eeb9f1516cc1417a064b5709629612149',
         'com.googlecode.jcsv:jcsv:73ca7d715e90c8d2c2635cc284543b038245a34f70790660ed590e157b8714a2',
         'com.github.sarxos:webcam-capture:d960b7ea8ec3ddf2df0725ef214c3fccc9699ea7772df37f544e1f8e4fd665f6',
+        'org.openjfx:javafx-fxml:ef5b10aa0b985e3bece34629bcaf4f462d3a60e1139a83edd116f69c1a36e0e8',
```

Comment:
I recall encountering issues before when we added the openjfx dependencies since they are OS specific. It may work for you on Mac, but will likely fail on Windows (and probably Linux).